shelley-ma/impl/cardano-ledger-shelley-ma.cabal

```diff
@@ -24,6 +24,8 @@ library
     Cardano.Ledger.Mary
     Cardano.Ledger.ShelleyMA.Value
     Cardano.Ledger.ShelleyMA.Timelocks
+    Cardano.Ledger.ShelleyMA.TxBody
+    -- Cardano.Ledger.ShelleyMA.TestTxBody
```

Comment:
I suppose this line only exists because this PR is WIP.

Reply:
Yes, I set up a new test suite inside cardano-ledger/shelley-ma/impl and moved all the tests ther.